lib/http2/scheduler.c

```diff
@@ -253,6 +254,7 @@ void h2o_http2_scheduler_relocate(h2o_http2_scheduler_openref_t *dst, h2o_http2_
     dst->_self_is_active = src->_self_is_active;
     dst->_queue_node._link = (h2o_linklist_t){NULL};
     dst->_queue_node._deficit = src->_queue_node._deficit;
+    dst->_is_stale = 1;
```

Comment:
I feel a bit uneasy here, because the name of the function (i.e. "relocate") or the signature does not imply that this function is making the reference "stale." We might see confusion in the future, if we modify a flag named as "stale" in a function that "relocates" an object.

That said, I understand that we might remove the bit as we change the way we detect if the client is Chromium. Or if that is not the case, I think we might be fine with having a "stale" flag as a public field in the object, letting the http2 stack directly modify the flag.

Reply:
Hmm, okay then how about renaming it to `is_relocated`?

> That said, I understand that we might remove the bit as we change the way we detect if the client is Chromium.

I don't think that's not the case, unfortunately. Regardless of the way we implement Chromium detection, I believe this flag is necessary anyway to implement `find_parent_by_weight` function correctly.